scripts/pipeline.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+PIPELINE='pipe_naf_reader'
```

Comment:
@smpiano check in other pipes how we are doing this. We always include the info in `pipe_naf_reader/__Init__.py` and then in this file we take the information from there. That way we only maintain the version in one place.